hrs-portlets-webapp/src/main/webapp/WEB-INF/tags/hrs/pagerNavBar.tag

```diff
@@ -9,15 +9,15 @@
 <div class="flc-pager-${position} dl-pager-navbar" style="display:none">
   <ul class="dl-pager-bar fl-pager-ui uw-pager-bar">
     <li class="flc-pager-previous dl-pager-previous">
-      <a href="javascript:;" title="Previous Page" class="btn btn-default">Previous</a>
+      <a href="#" title="Previous Page" class="btn btn-default">Previous</a>
```

Comment:
Ohhh yea, good call @timlevett. Discussing w/ @vertein and @jiayinjx now, it's probably not worth it to fix the accessibility on this. I take back my :+1: :(